lib/fluent/plugin/out_rdkafka.rb

```diff
@@ -91,8 +92,22 @@ def configure(conf)
     super
     log.instance_eval {
       def add(level, &block)
-        if block
+        return unless block
```

Comment:
[nits] Can we use `block_given?`?

Reply:
Does it mean change method args like below?

```
def add(level)
  return unless block_given?
```

Reply:
I thought just changing to `return unless block_given?` part. but yours also is fine for me.